src/printing.py

```diff
@@ -11,7 +11,11 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+rounded_y = round(y, 2)
+print("x is %s, y is %s, z is \"%s\"" % (x, rounded_y, z))
```

Comment:
This works, but there's also a way to do the rounding inside the formatted string itself. Check out this link for the first two problems https://mkaz.blog/code/python-string-format-cookbook/